docs/blog/2022-06-24-junopi/index.mdx

```diff
@@ -98,30 +109,114 @@ go version
 
 ### Install Juno ⚙️
 
-To install `juno`, the StarkNet node:
+### Installing
+
+Clone the repository:
 
 ```bash
-go install github.com/NethermindEth/juno/cmd/juno@latest
+git clone https://github.com/NethermindEth/juno
+```
+
+#### Install python dependencies
+
+We are currently only support python `3.7`, and we recommend use pyenv. To install it, you can follow this instruction:
+
+1. Install dependencies:
+
+```shell
+sudo apt-get update
+sudo apt-get install make build-essential git patch zlib1g-dev clang \
+  openssl libssl-dev libbz2-dev libreadline-dev libsqlite3-dev llvm \
+  libncursesw5-dev xz-utils tk-dev libxml2-dev libxmlsec1-dev libffi-dev \
+  liblzma-dev
 ```
 
-To install `juno-cli`, the separate tool for interacting with StarkNet:
+2. Install pyenv:
+
+```shell
+curl https://pyenv.run | bash
+```
+
+3. Add the following entries into your `~/.bashrc` file:
+
+```shell
+# pyenv
+export PATH="$HOME/.pyenv/bin:$PATH"
+eval "$(pyenv init --path)"
+eval "$(pyenv virtualenv-init -)"
+```
+
+4. Restart your shell:
+
+```shell
+exec $SHELL
+```
+
+5. Install python 3.7:
+
+```shell
+pyenv install 3.7.13
+pyenv global 3.7.13
+```
+
+6. Inside of the project folder, install python dependencies:
+
+```shell
+$ pip install -r requirenments.txt
+```
+
+#### Installing golang dependencies
+
+You can install all golang dependencies running the next command inside the project folder:
 
 ```bash
-go install github.com/NethermindEth/juno/cmd/juno-cli@latest
+$ go get ./...
 ```
 
-For details about the configuration you can look at
-the [config file description](https://gojuno.xyz/docs/running/config).
+### Compile
+
+```bash
+$ make juno
+```
+
+### Run
+
+To synchronize with the StarkNet state from the centralized feeder gateway, run the following
+command:
+
+```bash
+# For Ethereum Goerli testnet
+$ ./build/juno
 
+# For Ethereum Mainnet
+$ ./build/juno --netowrk 1
+```
+
+To sync the state without relying on the feeder gateway, configure an Ethereum node and run the following command:
+
+```bash
+# For Ethereum Goerli testnet
+$ ./build/juno --eth-node "<node-endpoint>"
+
+# For Ethereum Mainnet
+$ ./build/juno --netowrk 1 --eth-node "<node-endpoint>"
+```
+
+To view other available options please run `./build/juno -h`.
+
+For more configuration details, check the [config description](/docs/running/config).
+For details about the configuration you can look at
+the [config file description](/docs/running/config).
```

Comment:
Duplicate.

Reply:
Because the idea is that if you are in a tutorial you don't have to go to another page to see how to install what you need. I think it can be left in that way now.

Reply:
I don't understand. Isn't line 207 saying the same thing that's on lines 208-209 and also linking to the same destination?